app/src/main/res/values/strings.xml

```diff
@@ -9,9 +9,10 @@
     <string name="choose_direction_map">choose from map</string>
     <string name="choose_direction_search">search for location</string>
     <string name="choose_direction_current">use current location</string>
-    <string name="Confirm">Confirm</string>
-    <string name="Cancel">Cancel</string>
+    <string name="confirm">Confirm</string>
```

Comment:
Are we still using the string after switching to using "Go Here"?

Reply:
Checked their usages and they were never used in our project in the first place.

Reply:
I'm leaving them there in case they will be used in the future and changed their name/ID to lower caps to match conventions.